comixed-web/src/assets/i18n/fr/admin.json

```diff
@@ -1,28 +1,29 @@
 {
   "web-audit-log": {
     "clear": {
-      "confirmation-message": "Are you sure you want to clear the audit log? This action cannot be undone.",
-      "confirmation-title": "Clear Web Audit Log",
-      "effect-failure": "Failed to clear the web audit logs.",
-      "effect-success": "Web audit logs cleared."
+      "confirmation-message": "Êtes-vous sûr de vouloir vider le journal d'audit ? Cette action est irreversible.",
+      "confirmation-title": "Vider le journal d'audit Web",
+      "effect-failure": "Échec lors de la suppression du journal d'audit Web.",
+      "effect-success": "Journal d'audit Web vidé."
     },
     "label": {
       "actions": "Actions",
```

Comment:
Please translate this entry.

Reply:
Actions is translated as Actions it seems to me unless I don't understand its meaning correctly.

Reply:
No, it's more likely due to my weaker french.